### DISCUSSION TITLE
Welcome, Abhilash!

### DISCUSSION THREAD

**Discussion by dev1abhi** [ParanoidUser/yolo#1159] [Subscribe]
Leaving a message for the yolo badge, thanks.

**Comment by ParanoidUser** [comment]
Ahoy @dev1abhi, thank you for your interest in the project! To get a Yolo badge, simply follow the steps in https://github.com/ParanoidUser/yolo/discussions/18.

**Note:** Don't forget to assign a reviewer on step 4. 🔭

**Another note:** You should receive a repo collaborator invite that allows you to merge PRs (steps 5-6).

Let me know if you have any questions. Welcome aboard! 🚢⭐

**Reply by Yousran** [reply]
assign a reviewer pane doesn't show in PR. how to solve this?

**Comment by lucas-0liveira** [comment]
hi @dev1abhi
